client/campare.go

```diff
@@ -0,0 +1,124 @@
+package client
+
+import xlineapi "github.com/xline-kv/go-xline/api/xline"
+
+type CompareTarget int
+type CompareResult int
+
+const (
+	CompareVersion CompareTarget = iota
+	CompareCreated
+	CompareModified
+	CompareValue
+)
+
+type Cmp xlineapi.Compare
+
+func Compare(cmp Cmp, result string, v interface{}) Cmp {
+	var r xlineapi.Compare_CompareResult
+
+	switch result {
+	case "=":
+		r = xlineapi.Compare_EQUAL
+	case "!=":
+		r = xlineapi.Compare_NOT_EQUAL
+	case ">":
+		r = xlineapi.Compare_GREATER
+	case "<":
+		r = xlineapi.Compare_LESS
+	default:
```

Comment:
Please add a test case to cover the default branch.

Reply:
This part is to prevent users from using the api incorrectly. The current test is only planned to include the correct logic, the coverage of the error logic will be scheduled in the subsequent update.